sql/2019/17_CDN/17_01.sql

```diff
@@ -0,0 +1,19 @@
+#standardSQL
+# 17_01: Top CDNs used in the requests

```

Comment:
I've added a comment to #98 clarifying the metrics. For 01 and 02 let's use `WHERE firstHtml` to limit the requests only to pages whose HTML is served by a CDN.

```sql
#standardSQL
# 17_01: Top CDNs used per page
SELECT 
  client,
  cdn, 
  COUNT(*) AS freq,
  SUM(COUNT(*)) OVER(PARTITION BY client) AS total,
  ROUND(100 * (COUNT(*)/SUM(COUNT(*)) OVER(PARTITION BY client)), 2) AS pct
FROM (
  SELECT
    _TABLE_SUFFIX AS client,
    _cdn_provider AS cdn
  FROM
    `httparchive.summary_requests.2019_07_01_*`
  WHERE firstHtml
  ORDER BY
    client
)
GROUP BY client, cdn
ORDER BY client, freq DESC
```

Reply:
Corrected